avm/res/machine-learning-services/workspace/tests/e2e/ai/dependencies.bicep

```diff
@@ -0,0 +1,54 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Key Vault to create.')
+param keyVaultName string
```

Comment:
I know these are in the tests, but we've made these Optional in the main - do they need to stay required here? Or should they also be optional?